doc/ref_arch/kubernetes/chapters/chapter08.md

```diff
@@ -16,10 +16,66 @@ While this Reference Architecture is being developed, Gaps will be identified th
 <a name="8.2"></a>
 ## 8.2 Gap analysis
 
+- Container Run-Time Interfaces towards NFVI resources.
+- Multi-Tenancy
+- K8s as VM based VNF Orchestrator
+
+<a name="8.2.1"></a>
+### 8.2.1 Container run-time Interfaces towards NFVI resources
```

Comment:
Since Container Run-time Interface (CRI) is well-defined, I think in this context we need to use a different term.  Is this the OCI run-time spec?